include/game/enemies/mon.h

```diff
@@ -0,0 +1,19 @@
+#ifndef GUARD_ENEMY_MON_H
+#define GUARD_ENEMY_MON_H
+
+#include "global.h"
+#include "game/entity.h"
+
+typedef struct {
+    /* 0x00 */ SpriteBase base;
+    /* 0x0C */ Sprite s;
+    /* 0x3C */ s32 x;
+    /* 0x40 */ s32 y;
+    /* 0x44 */ s32 deltaY;
+    /* 0x48 */ s32 speedY;
+} Sprite_Mon; /* 0x4C */
+
+void initSprite_Enemy_Mon(MapEntity *me, u16 spriteRegionX, u16 spriteRegionY,
```

Comment:
Function is wrong

Reply:
Well, it's also wrong in Bell then! :D 
I'll have a look at the other headers.